samples/snippets/src/main/java/com/example/bigquery/CreateJob.java

```diff
@@ -52,17 +48,21 @@ public static void createJob(String query) {
               .setLabels(ImmutableMap.of("example-label", "example-value"))
               .build();
 
-      // The client libraries automatically generate a job ID.
-      // Override the generated ID with either the job_id_prefix or job_id parameters.
-      String jobId = "code_sample_" + UUID.randomUUID().toString().substring(0, 8);
-      Job job = bigquery.create(JobInfo.of(JobId.of(jobId), queryConfig));
-      job = job.waitFor();
-      if (job.isDone()) {
+      // The location and JobName must be specified; other fields can be auto-detected.
+      String jobName = "jobId_" + UUID.randomUUID().toString();
+      JobId jobId = JobId.newBuilder().setLocation("us").setJob(jobName).build();
+
+      // Create a job with job ID
+      bigquery.create(JobInfo.of(jobId, queryConfig));
```

Comment:
i believe jobId is optional right? we don't need to create our own. So I think we can link user to the documentation that explains this.